templates/step/edit.html.ep

```diff
@@ -140,16 +148,19 @@
                             </p>
                             <p>
                                 <!-- lists the specific properties -->
-                                <input type="checkbox" name="workaround" id="property_workaround"
-                                        %= 'checked' if $properties->{workaround}
-                                        >
+                                <input type="checkbox" name="workaround" id="property_workaround" >
                                 <label for="property_workaround">workaround</label>
                                 <%= help_popover('Help for the workaround property' =>
                                     'If a needle with the workaround property matches the match result is recorded as
                                     a \'soft-fail\'. It is suggested to mark the reason for this choice in either the
-                                    name of the needle (e.g. my-name-bsc1234-20161224) or with special tags')
+                                    name of the needle (e.g. my-name-bsc1234-20161224) or with special tags and
+									input the reason description in \'Description\' textbox.')
                                 %>
                             </p>
+							<div id="workaround_reason_div" style="margin-bottom:1em; display:none;">
```

Comment:
Use a `p` element instead of assigning `margin-bottom:1em;` manually here.

Reply:
May I keep it?  Although it does not need to set margin manually when using a p element,  I really like using a div. In my opinion, using a div looks more beautiful when the elements show or hide  depending on the situation.

Reply:
Is this about animations?

If more specific styling is desirable here I'd vote for a class instead of doing it in the `style` attribute.

Reply:
> Is this about animations?

I've just tested and there is no animation in both cases.

I would actually prefer to make it a span element and move it inside the previous p element. That would decrease the space between the workaround checkbox and the description which makes sense because they belong together. And you would get the margin for free and in consistency with other paragraphs. Of course you need to use `display: block` then but this is set via JavaScript anyways.

Reply:
I use a span element and move it inside the p, it works well.  But I do not understand your mean about display:block, I did not use it,  and span works well.

BTW: there is no animations, just show and hide.

Reply:
> BTW: there is no animations, just show and hide.

And that's sufficient :-)

> I did not use it, and span works well.

Ok, maybe the jQuery show() function uses it by default. But in general the span element would default to `display: inline` which wouldn't look nice here.

Reply:
> Ok, maybe the jQuery show() function uses it by default. But in general the span element would default to `display: inline` which wouldn't look nice here.
I understand your meaning. Actually I added a <br> before <span>, so I did not see abnormal view.  Is it proper?  
If do not use <br>,  I need to set display:block in javascript. 
In your opinion which way is better? :)

Reply:
I understand your meaning. Actually I added a <br> before <span>, so I did not see abnormal view.  Is it proper?  
If do not use <br>,  I need to set display:block in javascript. 
In your opinion which way is better? :)

Reply:
I understand your meaning. Actually I added a br before span element, so I did not see abnormal view.  Is it proper?  
If do not use br,  I need to set display:block in javascript. 
In your opinion which way is better? :)

Reply:
I prefer `display: block`. The `<br>` as you put it would always be displayed and not only if `#workaround_reason` is visible.